src/templates/login.ejs

```diff
@@ -6,8 +6,18 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Login - StatDriver</title>
+    <link rel="stylesheet" type="text/css" href="../assets/css/common.css">
+    <link rel="stylesheet" type="text/css" href="../assets/css/login.css">
 </head>
 <body>
-
+    <div id="rect">
+        <div id="logo">StatDriver</div>
+        <div id="information">was wir machen blablabla<br>weitere zeile blabla<br>noch eine zeile blabla</div><br>
```

Comment:
hier einfach noch einen kurzen hübschen text (z.B. aus dem README oder der dem Doku-Draft in der Cloud) einfügen.